packages/reason-css-parser/lib/Parser.re

```diff
@@ -1569,7 +1569,7 @@ and property_stroke_linejoin = [%value.rec "'miter' | 'round' | 'bevel'"]
 and property_stroke_miterlimit = [%value.rec "<number-one-or-greater>"]
 and property_stroke_opacity = [%value.rec "<alpha-value>"]
 and property_stroke_width = [%value.rec "<svg-length>"]
-and property_tab_size = [%value.rec "<integer> | <extended-length>"]
+and property_tab_size = [%value.rec " <number> | <extended-length>"]
```

Comment:
Why change from integer to number?

https://developer.mozilla.org/en-US/docs/Web/CSS/tab-size#values

Reply:
Formal syntax -- https://developer.mozilla.org/en-US/docs/Web/CSS/tab-size#formal_syntax